components/proxy/src/main/java/com/hotels/styx/metrics/reporting/jmx/JmxReporterService.java

```diff
@@ -46,15 +44,15 @@ public JmxReporterService(String domain, MeterRegistry registry) {
     public CompletableFuture<Void> start() {
         return CompletableFuture.runAsync(() -> {
             jmxMeterRegistry = new JmxMeterRegistry(key -> null, Clock.SYSTEM, nameMapper);
-            ((CompositeMeterRegistry) meterRegistry).add(jmxMeterRegistry);
+            meterRegistry.add(jmxMeterRegistry);
```

Comment:
Any idea why JMX needs its own registry? These metrics won't get reported to DD

Reply:
I don't know. This has been in Styx for a very long time. It may have been added before we had a clear idea of what to implement.